.github/ISSUE_TEMPLATE/tf2_bug_report.md

```diff
@@ -0,0 +1,55 @@
+---
+name: TF2 bug report
+about: A bug in vanilla TF2 that could be fixed in Team Comtress
+title:
+labels: bug, tf2 issue, needs triage
+assignees:
+
+---
+
+### Description
+<!--
+A clear and concise description of what the bug is.
+-->
+
+### Checklist
+<!-- You do not have to answer "yes" to all of these to open an issue. -->
+- [ ] None of the *open or closed* issues document this problem.
+- [ ] This is a bug, not a new idea.
+- [ ] This is obviously a bug, not something that people commonly disagree on.
+- [ ] This bug is in the code of the game, not in the textures, sounds, or other resources.
+- [ ] This issue occurs on the lastest vanilla TF2 update.
+- [ ] This issue occurs on the latest Team Comtress release, or the `master` branch.
+- [ ] This issue occurs when using no custom content.
+- [ ] This issue occurs when using `start_tf2.bat` (or `start_server.bat`) with no custom launch options.
+- [ ] This issue occurs when using a server started with the `map` command from the game.
+- [ ] This issue occurs when using a dedicated server.
+- [ ] This bug is in the internal procedures or tooling (e.g. the build process).
+
```

Comment:
maybe add [ ] This issue exists on official bugtracker (and link, im too lazy to add)

and place to tag it somewhere it can contain more info.